docs/source/tutorial/create_a_pipeline.md

```diff
@@ -149,7 +149,7 @@ You should see output similar to the below:
 You can run the `preprocess_shuttles` node similarly. To test both nodes together as the complete data processing pipeline:
 
 ```bash
-kedro run
+kedro run --nodes=preprocess_companies_node,preprocess_shuttles_node
```

Comment:
I am not a massive fan of this change. I see what you're doing but it complicates things. At this point, it's fine to use just `kedro run` to execute the full data processing pipeline. Sure, later you'll have to use node names or pipeline name to single it out, but at this point it's a simplistic example with 2 nodes for new starters. `kedro run` is fine.

If you do want to include this change, you'll need to do it later down the page where the example runs the pipeline again, otherwise it's inconsistent.